public_html/admin-v2/races/js/results.js

```diff
@@ -29,31 +29,48 @@ const results = new Vue({
     availableWinHorses: function () {
       const placeId = this.enteredResults.place;
       const showId = this.enteredResults.show;
-      return this.horses.filter(horse => horse.id != placeId && horse.id != showId);
+      return this.horses.filter(
```

Comment:
Can we encapsulate this logic and reuse it? Just to have a single place of truth.